pkg/vct/logmonitoring/resthandler/activate.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package resthandler
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/trustbloc/edge-core/pkg/log"
+	"github.com/trustbloc/sidetree-core-go/pkg/restapi/common"
+)
+
+const (
+	activateEndpoint = "/activate-log"
```

Comment:
I wonder if we should have a single /log endpoint with a JSON payload that can determine the action?

Reply:
Do you mean something like:
{
  "action": "activate",
  "log-url": "https://vct.com/log"
}